admin_app/src/components/NavBar.tsx

```diff
@@ -37,21 +46,18 @@ export const NavBar = () => {
             />
           </svg>
         </button>
-        <div className="hidden w-full md:block md:w-auto" id="navbar-default">
+        <div
```

Comment:
Would be nice if (a) the menu icon shows that it is open (b) the menu floats on top of cards etc. and not take up space. But maybe these are for when we do styling with material-ui?

![image](https://github.com/IDinsight/aaq-core/assets/14125957/f2152675-ccfd-4512-804a-426a73613abc)

Reply:
Fixed (a). Could not fix (b) quick enough. I guess we can wait for the material refactor.